src/arkit.cs

```diff
@@ -2086,4 +2202,103 @@ interface ARMeshGeometry : NSSecureCoding {
 		[NullAllowed]
 		ARGeometrySource Classification { get; }
 	}
+
+	[iOS (14, 0)]
+	[BaseType (typeof (NSObject))]
```

Comment:
`[DisableDefaultCtor]` as this is only exposed in a getter so there's no reason to be able to create one (if it even works)